openapi.yaml

```diff
@@ -2165,6 +2177,634 @@ paths:
       security:
         - oscal_auth:
             - read:parties
+  /plan-of-action-and-milestones:
+    get:
+      tags:
+        - OSCAL Plan of Action and Milestones
+      summary: Returns all OSCAL Plan of Action and Milestones
```

Comment:
@brian-easyd Can you grammar check here? "OSCAL Plan of Action and Milestones" or "OSCAL Plans of Action and Milestones" (like "attorneys general")?

Reply:
Based on https://pages.nist.gov/OSCAL/reference/latest/complete/json-outline/, NIST has it as OSCAL Plan of Action and Milestones. But agree `Plans` may read better, especially on line `2184`.

Reply:
Yeah but I think that situation, it's talking about a single POA&M (hence, `poam-items` as an attribute). If we're returning multiple POA&Ms then are there Plans?

Reply:
@kylelaker this is correct as-is. When expanding the acronym, it officially has "Plan" as singular and "Milestones" as plural. The POA&M -is- the one and only plan for resolving vulnerabilities within a given system. It contains multiple milestones toward achieving that plan.